inverse_thomson_scattering/runner.py

```diff
@@ -142,6 +143,13 @@ def calc_spec(config):
     config["other"]["lamrangI"] = [524, 529]
```

Comment:
do these need to be hardcoded? ideally, this should follow the same code path as the fitting but just perform the forward pass once except for where it can't

Reply:
these parameters are determined from the data, but in the next update i'll spec them relative the the fit ranges and maybe we should add an independent field for them in the future